doc-content/enterprise-only/openshift/template-deploy-ha-resources-proc.adoc

```diff
@@ -0,0 +1,24 @@
+[id='template-deploy-ha-resources-{context}-proc']
+= Configuring resource usage for a high-availability deployment
+
+If you are deploying the high-availability template (`{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-authoring-ha.yaml`), you can optionally configure resource usage to optimize performance for your requirements.
+
+If you are deploying the single authoring environment template (`{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-authoring.yaml`), skip this procedure.
+
+.Prerequisites
+
+* You started the configuration of the template, as described in {startlink}.
+
+.Procedure
+
+Set any of the following parameters of the template as necessary:
```

Comment:
Set all applicable parameters of the template as follows:

Reply:
I think your formula might read, especially to a non-native speaker, as requiring one to set all the parameters, which is what I am explicitly trying to avoid.

Reply:
Remember the 80/20 rule. We have to make most sense to customers who are used to reading technical documentation. Obfuscating actually makes translations harder.